plugins/source/aws/resources/services/efs/filesystems.go

```diff
@@ -72,7 +77,7 @@ func ResolveEfsFilesystemBackupPolicyStatus(ctx context.Context, meta schema.Cli
 	})
 	if err != nil {
 		if cl.IsNotFoundError(err) {
-			return resource.Set(c.Name, types.StatusDisabled)
+			return nil
```

Comment:
Lets not change this in this pr because then we will have to mark the next release as a breaking change. I have added that [change to our list of](https://github.com/cloudquery/cloudquery/issues/12648) changes to include in the next breaking change (which is tbd)